packages/next/src/server/base-server.ts

```diff
@@ -2484,10 +2484,8 @@ export default abstract class Server<
         query: origQuery,
       })
 
-      const shouldWaitOnAllReady =
-        !supportsDynamicResponse ||
```

Comment:
I'm worried about this change because it seems like there are a lot of branches that set `supportsDynamicResponse`. 

Would it make more sense to ensure `supportsDynamicResponse` is false in the case of `isHTMLBot && isRoutePPREnabled`?

Reply:
We have few places might change the `supportsDynamicResponse` value before passing down to app-render.

But we're also still checking it in app-render here, so it's fine to remove the one here and just relying on the final accumulation in app-render
https://github.com/vercel/next.js/blob/61b81f4fddffc043e08099743fefc02773cea941/packages/next/src/server/app-render/app-render.tsx#L1997-L1999